lib/ansible/modules/network/aci/aci_aaa_user.py

```diff
@@ -277,6 +283,7 @@ def main():
         last_name=dict(type='str'),
         phone=dict(type='str'),
         state=dict(type='str', default='present', choices=['absent', 'present', 'query']),
+        name_alias=dict(type='str', aliases=['nameAlias_name', 'alias']),
```

Comment:
Why add 2 aliases for this? This is new functionality, we do not add aliases for new functionality.
Parameter aliases are intended to provide a smooth upgrade path from an (incorrect) name to a better name.
Why would we add 2 incorrect names to new parameters? It would simply add confusion.

Reply:
@nkatarmal-crest please remove the 2 diff aliases and keep (type='str').